sidechain/consensus/aura/src/verifier.rs

```diff
@@ -134,15 +143,20 @@ where
 	ensure!(
 		expected_author == block.block_data().block_author(),
 		ConsensusError::InvalidAuthority(format!(
-			"Expected author: {:?}, author found in block: {:?}",
-			expected_author,
-			block.block_data().block_author()
+			"Expected author: {}, author found in block: {}",
+			to_string(expected_author),
+			to_string(block.block_data().block_author())
```

Comment:
Great! But why not implement the `ToString` trait directly? If this is not possible, I'd love to see a small comment, why the below function exists.

Reply:
I wanted to implement `Debug` or `Display` for that type. But since that type is a substrate type, not under our control or crate, I cannot implement a trait for it? So that's why I was left with this (not so nice) option.

Reply:
Ah yeah, of course, I kind of thought it is the same as with the `From<T>` type, but yeah, as the trait is not parametrized, we can of course not implement the external trait for an external type. 👍

Reply:
No comment needed, leave it as is in this case.